roles/cloud-linode/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Build python virtual environment
+  import_tasks: venv.yml
+
+- name: Include prompts
+  import_tasks: prompts.yml
+
+- name: "Creating an instance..."
+  linode_v4:
+    access_token: "{{ algo_linode_token }}"
+    label: "{{ algo_server_name }}"
+    state: present
+    region: "{{ algo_linode_region }}"
+    image: "{{ cloud_providers.linode.image }}"
+    type: "{{ cloud_providers.linode.type }}"
+    authorized_keys: "{{ public_key }}"
+    root_pass: "{{ algo_linode_password }}"
```

Comment:
We don't need to specify this, the module will generate it automatically, which I'm not a fan of either. We would need to disable root login by password for security reasons.

Reply:
Linode doesn't support used_data in cloud-init thus we have to use the common role to set  `PermitRootLogin prohibit-password` in /etc/ssh/sshd_config

Reply:
Made these changes in https://github.com/trailofbits/algo/pull/1590/commits/2ac6189ae7a76fbd8cf3f608a9455532b69770f1 and confirmed with a test deploy. I'm a bit of an Ansible noob, though, so let me know if there's a more idiomatic way to do them/place to put them.